Cargo.toml

```
@@ -14,7 +14,8 @@ edition = "2021"
 autoexamples = false
 
 # also update in README.md (badge and "Rust version requirements" section)
-rust-version = "1.56"
+# 1.57.0 - Now using '.as_slice()'
+rust-version = "1.57.0"
```

```suggestion
rust-version = "1.65.0"
```

The new MSRV is actually 1.65, because nom 8 will use GATs